registry_module.go

```diff
@@ -696,6 +713,15 @@ func (o RegistryModuleCreateWithVCSConnectionOptions) valid() error {
 	if o.VCSRepo == nil {
 		return ErrRequiredVCSRepo
 	}
+
+	if o.TestConfig != nil {
+		if *o.TestConfig.TestsEnabled {
```

Comment:
Unless I am mistaken, I think it's possible for this field to be nil. I think that would cause a nil pointer dereference panic.

Reply:
Good catch. Yeah, this needs a nil check please!

Reply:
addressed!